### ISSUE TITLE
Accessibility: "Broken ARIA menu"

### ISSUE THREAD

**Issue by MarkoTP** [microsoft-search/pnp-modern-search#4028]
**Version used**
4.13.1

**Describe the bug**
Accessibility:
A few PnP Modern Search issues were identified during the accessibility audit.
Missing elements in ARIA menu in PnP Modern Search Filter webpart. Technical issue from Wave accessibility evaluation tool: "Broken ARIA menu" in filter category title.
Wave accessibility evaluation tool: https://webaim.org/

Wave tool reference:
> What It Means
> An ARIA menu does not contain required menu items.
> 
> Why It Matters
> ARIA menus are application menus (like those used in software menu) with a specific keyboard interactions. They are NOT for navigation links on a web site and must contain at least one menuitem, menuitemcheckbox, or menuitemradio element.
> 
> How to Fix It
> Ensure that the menu is an application menu and has the appropriate keyboard interactions (menu items are navigated via the arrow keys, not via the Tab key) and internal menu items, otherwise remove the menu role.
> 
> The Algorithm... in English
> An element with role="menu" does not contain at least one element with role="menuitem", role="menuitemcheckbox", or role="menuitemradio".

WCAG 2.1, 2.1.1, 4.1.2

**To Reproduce**
Detailed steps to reproduce the behavior:
Configure search page you like with search box, results and filters web parts. There should be at least one filter defined.
Execute a search with keyword that returns filterable results.
Execute the Wave accessibility tool on the page.
The Wave tool should find "Broken ARIA menu" -errors in filter category title and provide suggestions for fixing them.

**Expected behavior**
The Wave tool should not find any technical errors in the web part.

**Desktop (please complete the following information):**
Browser: Chrome, Version 129.0.6668.90 (Official Build) (64-bit)

**Comment by kasperbolarsen** [comment]
Please be aware that we are not obliged to implement WCAG 2, and I doubt that any of the developers that are contributing to the project will prioritize these issues. Is there any chance that you or somebody working on your behalf will provide the PRs?
